server/src/routes/common/centre-controllers.js

```diff
@@ -262,3 +265,14 @@ export async function createCentre (req, res) {
     })
   }
 }
+
+export async function getCentresByDepartement (req, res) {
+  const { departementId } = req.query
+  const deptCenters = await findCentresByDepartement(departementId)
+  appLogger.info({
+    description: 'Getting candidat centers associated to Paris',
+    section: 'candidat-deptCenters',
+    deptCenters,
+  })
+  res.json({ success: true, deptCenters })
```

Comment:
doit-on envoyer tous les informations des centres?

Reply:
done in PR #923